scripts/verify-phase-52-6-6-root-package-guardrails.sh

```
@@ -0,0 +1,173 @@
+set -euo pipefail
+
+repo_root="${1:-$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)}"
+doc_path="${repo_root}/docs/adr/0014-phase-52-6-1-root-shim-inventory-and-deprecation-contract.md"
+package_root="${repo_root}/control-plane/aegisops_control_plane"
+
+required_phrases=(
+  'After Phase 52.6.6, the only retained root owner files are'
+  '`__init__.py`, `config.py`, `models.py`, `operator_inspection.py`, `persistence_lifecycle.py`, `publishable_paths.py`, `record_validation.py`, `reviewed_slice_policy.py`, `service_composition.py`, and `structured_events.py`.'
+  '`service.py` is not a retained owner; it is the single retained compatibility blocker'
+  'No other direct root Python file may be promoted to retained owner status without a later accepted ADR or issue-specific contract'
+  'The root package guardrail baseline is exactly `37` direct `.py` files under `control-plane/aegisops_control_plane/`.'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)